druntime/src/core/sys/linux/input.d

```diff
@@ -0,0 +1,236 @@
+module core.sys.linux.input;
+
+version (linux):
+extern(C):
+nothrow:
```

Comment:
Nit: Why only `nothrow`? I would make it `@nogc` as well, but then again there aren't any functions in here, so it might as well be neither.

Reply:
I don't know, most modules in `core.sys.linux` have these for some reason.

Then again, there are no function declarations or function pointers in none of these modules, no `@nogc` would do nothing here. But, neither does `nothrow`.